Gopkg.toml

```diff
@@ -22,33 +22,33 @@
 
 required = ["k8s.io/code-generator/cmd/client-gen"]
 
-[[constraint]]
-  name = "github.com/stretchr/testify"
+#[[constraint]]
+#  name = "github.com/stretchr/testify"
 
-[[constraint]]
+[[override]]
```

Comment:
is override needed because it's a transitive dep?

Reply:
correct, `dep` showed a warning when using `constraint` since it wasn't used directly in the repo